SpatialGDK/Source/SpatialGDK/Private/Interop/SpatialReceiver.cpp

```diff
@@ -1072,7 +1070,7 @@ void USpatialReceiver::DestroyActor(AActor* Actor, Worker_EntityId EntityId)
 	check(PackageMap->GetObjectFromEntityId(EntityId) == nullptr);
 }
 
-AActor* USpatialReceiver::TryGetOrCreateActor(UnrealMetadata* UnrealMetadataComp, SpawnData* SpawnDataComp)
+AActor* USpatialReceiver::TryGetOrCreateActor(UnrealMetadata* UnrealMetadataComp, SpawnData* SpawnDataComp, SpatialGDK::ComponentPresence* ComponentPresenceData)
```

Comment:
I think we discussed this in passing last week, but remind me again why non-auth clients don't need the component presence component (re: REQUIRED_COMPONENTS_FOR_NON_AUTH_CLIENT_INTEREST?)

Reply:
Oh, it's only currently used by servers when creating controllers. If this component were used for entity completeness, they'd need it so I wonder if it's simpler to just include it now. Otherwise we need to be careful the ComponentPresence* obj is never null when dereferencing.

Reply:
Not sure including for entity completeness on clients is a sensible line of reasoning? I'd assuming there are more components that a non-auth server should see on an Actor entity vs. a non-auth client beyond the NetOwnerClientId one? I feel like when we come to do completeness for clients this we'd add an additional list for non-auth clients but should maybe leave as it is for now?